src/code/sys_math3d.c

```diff
@@ -29,6 +29,9 @@ struct Dummy224 { int x; };
 struct Dummy225 { int x; };
 struct Dummy226 { int x; };
 struct Dummy227 { int x; };
+struct Dummy228 { int x; };
+typedef int Dummy229;
+
 // clang-format on
 
 // Headers are currently valued at 63 mod 256 (./tools/calc_bss.sh <headers>)
```

Comment:
```suggestion
// Headers are currently valued at 60 mod 256 (./tools/calc_bss.sh <headers>)
```
(maybe double-check this quickly by running calc_bss.sh on the headers, but this is what it should be to get the same bss index)